ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml

```diff
@@ -23,6 +23,7 @@ workflow:
     allow_best_effort_post_steps: true
     allow_skip_on_success: true
     post:
+    - ref: hypershift-kubevirt-baremetalds-disconnected-highlightmissingimages
```

Comment:
AFAIK, hypershift-mce-dump can accomplish the same thing.
E.g: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/54062/rehearse-54062-periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected/1812774648182476800/artifacts/e2e-kubevirt-metal-mce-disconnected/hypershift-mce-dump/artifacts/hostedcluster-6d047c994a7bcc9e751e/

Reply:
I'm also adding it to `hypershift-kubevirt-baremetalds-disconnected-conformance-workflow.yaml` which is not calling `hypershift-mce-dump`, but more than that is a matter of visibility: for sure we already have all the info we need to debug an issue in the must-gather archives we are already collecting, but getting at least one of the images not correctly mirrored on the internal registry is by far the most common issue in those disconnected lanes and this new step will `highlight` it if the issue is really there.

Reply:
okay, Thanks